src/Routes/auth-login/auth-login.ts

```diff
@@ -0,0 +1,171 @@
+require("dotenv").config();
+const { Router } = require("express");
+const session = require("express-session");
+const bcrypt = require("bcrypt");
+const loginauth = Router();
+const jwt = require("jsonwebtoken");
+
+
+
+const {SESS_NAME = "sessionid",JWT_ACCESS_KEY="shivam"} = process.env
+
+const Users = [
+  {id:1,name:"shivam",email:"shivam@gmail.com",password:'password'},{id:2,name:"shiva",email:"shiva@gmail.com",password:'password'},{id:3,name:"pandey",email:"pandey@gmail.com",password:'password'},{id:4,name:"aditya",email:"aditya@gmail.com",password:'password'}
+]
+
+
+
+const authenicatelogin = (req: { session: any; },res: any,next: any) => {
```

Comment:
middlewares should be in their own folder

Reply:
for middlewares can I create a separate directory?